code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -323,6 +323,83 @@
 	..()
 	. = 1
 
+/datum/reagent/consumable/tea/red
+	name = "Red Tea"
+	description = "Tasty red tea, it has digestive properties. Drink in moderation!"
```

Comment:
'digestive properties' sounds off because it's not like the tea is acidic, it does not have the properties of a digestive system, it _helps_ digestion